openstack/nova/templates/placement-db-split-job.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if and .Values.placement.enabled (not .Values.placement.unsplit) .Values.placementdbUser .Values.placementdbPassword .Values.placementdbName }}
+{{- /* See: https://github.com/openstack/placement/blob/master/placement_db_tools/mysql-migrate-db.sh */}}
+{{- $envAll := . }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-placement-db-split"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ default "unknown" .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}-placement-db-split"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ .Release.Name }}-pod-exec
+      containers:
+      - name: placement-db-split
+        image: "{{ required ".Values.global.dockerHubMirrorAlternateRegion is missing" .Values.global.dockerHubMirrorAlternateRegion }}/{{ .Values.pre_change_hook.image }}:{{ .Values.pre_change_hook.image_version }}"
```

Comment:
Why are we using the alternate region's dockerhub mirror here?

Reply:
Mostly because I copied and pasted it. I changed it to `.Values.global.registry`